index.html

```diff
@@ -209,5 +209,18 @@ <h2 id="heading-2">Skills</h2>
       <hr>
     </section>
 </main>
+<section class="form">
+  <div class="about-me">
+    <h1 class="heading1"  >Contact me</h1>
+    <P class="para-1" >If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</P>
+  </div>
+  <form action="https://formspree.io/f/xeqwglky" method="POST">
+    <input type="text" value="Diana"  maxlength="30"  placeholder="Name" class="input text" required/>
+    <input type="email" value="dianab21@gmail.com" placeholder="Email" class="input text" required/>
+    <textarea id="message" cols="30"  maxlength="500"   rows="9" placeholder="Write Your Message Here"></textarea> 
+
```

Comment:
Kindly make sure that the textarea has a required attribute because according to the requirement, all input fields must have the required attribute to activate the validation on submit 😊 ☺️